packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -259,6 +272,26 @@ export class TableColumnDateText extends TableColumnTextBase {
         };
         return options;
     }
+
+    private validateConfig(formatter: Intl.DateTimeFormat | undefined): void {
```

Comment:
It doesn't look like you're actually using `formatter` in this function other than to determine if the configuration is valid (`formatter !== undefined`) or invalid (`formatter === undefined`). I think the intent of this function would be clearer if you pass a boolean rather than the formatter.